### DISCUSSION TITLE
Allow sharing of training data

### DISCUSSION THREAD

**Discussion by osi1880vr** [civitai/civitai#124] [Ideas]
There is a load of models doing the same or nearly the same, now we see them merged to get together what they do good. Why not alllow to share training data so people could use that to directly train models to allow even more awesome results.

Just thinking loud =)

**Comment by JustMaier** [comment]
We've had this in place since launch, but it wasn't obvious how to do it. We just released a new File Upload UI last week that makes it much more intuitive:
![image](https://user-images.githubusercontent.com/607609/211394980-b5dade9b-4bf5-4c3b-a48e-8bd9e5141429.png)

Just need to hit "+ Add Training Data" and it will add a file input for you to include your training data.
